packages/front-end/components/Settings/DataSources.tsx

```diff
@@ -25,6 +25,10 @@ const DataSources: FC = () => {
     mutateDefinitions,
     ready,
   } = useDefinitions();
+  const filteredDatasources = datasources.filter((ds) => {
+    if (!project) return true;
+    return ds?.projects?.includes(project);
```

Comment:
Is this right?  If a data source is not tagged to any projects, it should always be included, right?